pkg/pubsub/adapter/adapter_test.go

```diff
@@ -197,9 +196,6 @@ func TestInboundConvert(t *testing.T) {
 }
 
 func TestReceive(t *testing.T) {
-	traceID := "1234567890abcdef1234567890abcdef"
```

Comment:
fyi @ian-mi @nachocano Had to ignore these fields to get tests passing. Let me know if there is a better alternative.

Reply:
maybe just comment this out instead of removing and add a TODO to the issue will or has created on the SDK?

Reply:
done